sdk/python/feast/loaders/file.py

```diff
@@ -66,47 +69,34 @@ def export_source_to_staging_location(
         uri_path = None  # type: Optional[str]
         if uri.scheme == "file":
```

Comment:
Can the handling of file:/// type URLs be abstracted away inside the StorageClient via LocalFSProtocol?

Reply:
@mrzzy I tried while refactoring this file. This particular case didn't fit well with the abstraction. This part of the code is saving the data frame into a temp location and then it will be uploaded into a staging location. I saw those two as separate concerns as staging location itself could be a local file system.

Reply:
@jmelinav makes sense if we see in a staging protocol perspective using the current abstraction. See comment on `StagingProtocol`. If we consider this as a storage client/backend, integrating this might make more sense.